docs/miscellaneous.rst

```diff
@@ -37,15 +37,18 @@ The elements of structs and arrays are stored after each other, just as if they
 Due to their unpredictable size, mapping and dynamically-sized array types use a Keccak-256 hash
 computation to find the starting position of the value or the array data. These starting positions are always full stack slots.
 
+Mappings and Dynamic Arrays
+===========================
+
 The mapping or the dynamic array itself
 occupies an (unfilled) slot in storage at some position ``p`` according to the above rule (or by
```

Comment:
unfilled is only true for mappings

Reply:
Ok. I'll remove, since we're explaining it in more detail anyways.